.github/scripts/clean-build.sh

```diff
@@ -201,7 +189,6 @@ rm -f $(find config -type f -name '*.ini.php' -not -name global.ini.php)
 rm -f config/config.php
 
 rm -rf tmp/*
-rm -f HIRING.md
```

Comment:
I'd probably keep this one here as if we add the file in future, we don't have to think about re-adding it here again.

Reply:
I doubt we would readd that file. It was last used like 5 years ago and actually existed for over 3 years, even though the role was already closed, as we forgot to remove it again 🙈